Command/EncryptPasswordCommand.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the ekino/data-protection-bundle project.
+ *
+ * (c) Ekino
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Ekino\DataProtectionBundle\Command;
+
+use Ekino\DataProtectionBundle\DependencyInjection\Configuration;
+use Ekino\DataProtectionBundle\Encryptor\Encryptor;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+/**
+ * Class EncryptPasswordCommand
+ *
+ * @author Christian Kollross <christian.kollross@ekino.com>
+ */
+class EncryptPasswordCommand extends Command
+{
+    /**
+     * {@inheritdoc}
+     */
+    protected static $defaultName = 'ekino-data-protection:encrypt';
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this->setDescription('Encrypt a short text (like a password) with a given secret')
+            ->addArgument('text', null, 'Text to encrypt')
+            ->addArgument('secret', null, 'Application secret')
```

Comment:
Should be an option, and by default it will fetch the project secret (of the current env)

Reply:
Done.